redirects.yaml

```diff
@@ -42,6 +42,30 @@ redirects:
 - from: /docs/webstore/pricing/
   to: /docs/webstore/money/
 
+# CWS payments/apps purge
+- from: /docs/webstore/hosted_apps
+  to: /docs/extensions/webstore
+- from: /docs/webstore/migrating
+  to: /docs/extensions/webstore
+- from: /docs/webstore/get_started_simple
+  to: /docs/extensions/webstore/publish/
+- from: /docs/webstore/apps_vs_extensions
+  to: /docs/extensions/webstore
+- from: /docs/webstore/distribution
+  to: /docs/extensions/webstore
+- from: /docs/webstore/launching
+  to: /docs/extensions/webstore
+- from: /docs/webstore/choosing
+  to: /docs/extensions/webstore
+- from: /docs/webstore/samples
+  to: /docs/extensions/webstore
+- from: /docs/webstore/get_started
+  to: /docs/extensions/webstore
+- from: /docs/webstore/inline_installation
+  to: /docs/extensions/webstore
+- from: /docs/webstore/authentication
+  to: /docs/extensions/webstore
+
```

Comment:
I noticed that both `/docs/webstore/one_time_payments` and `/docs/webstore/payments-iap/index.md` don't have redirects. Is this omission intentional? If so, what is the rationale behind not redirecting these pages?

Reply:
Not intentional, just an oversight. Fixed. Note that I opened #898 to look for other links broken by this or other purges.